_posts/2020-10-15-september-rubygems-update.md

```diff
@@ -0,0 +1,67 @@
+* Fixed Ruby `setup.rb` unnecessarily [rewriting the bundler gemspec](https://github.com/rubygems/rubygems/pull/3980).
+* Fixed situations where [bundler would crash if running on a path including brackets.](https://github.com/rubygems/rubygems/pull/3854)
+* We've [shipped the `bundle fund` command](https://github.com/rubygems/rubygems/pull/3390) that lists out all the URLs for `gems` whose maintainers are actively looking for funding.
+* Responded to HackerOne reports for RubyGems.
+* Merged a PR [adding docs about Gemstash.](https://github.com/rubygems/guides/pull/266)
+* [Updated docs to recommend Gemstash instead of gem server.](https://github.com/rubygems/guides/pull/269)
+* Added a note about [credentials in the rubygems.org repo being fake, in response to a HackerOne report.]( https://github.com/rubygems/rubygems.org/pull/2530)
+* Caught up with changes in `ruby-core` regarding versioning default gems. ([#3937](https://github.com/rubygems/rubygems/pull/3937) and [#3938](https://github.com/rubygems/rubygems/pull/3938))
+* Fixed [issue with configuration priority.](https://github.com/rubygems/rubygems/pull/3933)
+* [Deprecated  bundle cache --all.](https://github.com/rubygems/rubygems/pull/3932)
+* Created a unified release & changelog management workflow.
+* Worked on breaking [#3317](https://github.com/rubygems/rubygems/issues/3317) into multiple more actionable issues ([#3317](https://github.com/rubygems/rubygems/issues/3317#issuecomment-692449034) and [#3957](https://github.com/rubygems/rubygems/issues/3957))
+* Fixed some issues with CI to adapt to `ruby-core` changes.
+
+In addition, we worked on other fixes including enabling disable_multisource and figuring out the different new behaviours it enables and a bunch of other `test/dev` issues and reviewed PRs from external contributors.
+
+In September, Rubygems gained [150 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2020-09-01%7D...master@%7B2020-09-30%7D), contributed by 12 authors. There were 1263 additions and 4300 deletions across 176 files.
+
+## RubyGems.org News
+In September, RubyGems.org saw some changes 
+* imported [fastly rubygems.org production configuration to terraform](https://github.com/rubygems/rubygems-terraform/pull/4)
+* [updated fastly `vcl` to unset `X-Forwarded-Host` from requests to fix a HO report.](https://github.com/rubygems/rubygems-terraform/pull/5)
+* [added regex whitelist for url on honeycomb logs export](https://github.com/rubygems/rubygems-terraform/pull/7)  to ensure we don't inadvertently send any sensitive info.
+* bulk update rubygems downloads count to [reduce 95th percentile processing time of FastlyLog Processor up to 40 sec.](https://github.com/rubygems/rubygems.org/pull/2527)
+* [fixed total count show in search pagination.](https://github.com/rubygems/rubygems.org/pull/2526)
```

Comment:
line 39-43 items were done in October. it would nicer to talk about these in the next update post.

Reply:
also, please adding following item:
```
* deprecated [help.rubygems.org](https://help.rubygems.org/) in favor of [support@ruubygems.org](mailto:support@rubygmes.org) to resolve the issue of genuine help tickets being marked as spam.
```